addons/csi-kubevirt/2-kubevirt-csi-controller.yaml

```diff
@@ -72,21 +72,6 @@ roleRef:
   name: kubevirt-csi-controller-cr
   apiGroup: rbac.authorization.k8s.io
 ---
-{{ with .Config.CloudProvider.Kubevirt -}}
-{{ with .InfraClusterKubeconfig }}
-apiVersion: v1
-data:
-    value: "{{ . }}"
-kind: Secret
-metadata:
-  labels:
-    app: kubevirt-csi-driver
-  name: infra-kubeconfig
-  namespace: kube-system
-type: Opaque
----
-{{ end }}
-{{ end }}
```

Comment:
This secret it shared between CSI and CCM, so naturally it's not managed by the addons but rather by kubeone itself.